gnosis/src/main/scala/edu/uci/ics/cloudberry/gnosis/NewYorkGnosis.scala

```diff
@@ -5,13 +5,13 @@ import java.io.{File, FilenameFilter}
 import com.vividsolutions.jts.geom.{Coordinate, Envelope, Geometry}
 import play.api.libs.json.{JsObject, Json, Writes}
 
-class NewYorkGnosis(levelGeoPathMap: Map[TypeLevel, File]) {
+class NewYorkGnosis(levelGeoPathMap: Map[TypeLevel, File]) extends IGnosis{
 
   import NewYorkGnosis._
 
-  val levelShapeMap: Map[TypeLevel, NYGeoJSONIndex] = load(levelGeoPathMap)
+  lazy val levelShapeMap = load(levelGeoPathMap)
```

Comment:
It's better to be not lazy, because when we start the server, we want to load everything and then begin to response. Otherwise the first query will be too slow.